src/monattack.cpp

```diff
@@ -2962,16 +2985,27 @@ bool mattack::grab_drag( monster *z )
         return false;
     }
 
+    const optional_vpart_position veh_part = get_map().veh_at( target->pos() );
+    if( foe->in_vehicle && veh_part ) {
+        const cata::optional<vpart_reference> vp_seatbelt = veh_part.avail_part_with_feature( "SEATBELT" );
+        if( vp_seatbelt ) {
+            target->add_msg_player_or_npc( m_good, _( "%1s tries to drag you, but is stopped by your %2s!" ),
```

Comment:
You are using three different colors for the same action, `m_good` (green), `m_info` (blue) and `m_warning` (yellow) as far as I can see. I would get rid of one of them, potentially `m_good`. Being dragged unsuccessfully is still most definitely a cause of concern because there is danger nearby, so neutral (`m_info`) seems the most appropriate here.

Reply:
There is `m_mixed` exactly for these kind of situations.

Reply:
Good point.

Reply:
I've actually used these message spans due to the existing messages for the attacks using them as well; they're kinda all over the place in normal `grab_drag` too. Should I change em there too?